lib/data_sources/ht_collections.rb

```diff
@@ -50,11 +56,21 @@ def load_from_db
         .transform_values { |h| DataSources::HTCollection.new(**h) }
     end
 
-    # TODO - should reload periodically -- when a new job starts, at least.
-    # Re-checking the database on a miss might also be sufficient.
-    def [](collection)
+    def load_data(collections)
+      @collections = data_from_db
+      @cache_timestamp = Time.now.to_i
+    end
+
+    def collection_info(collection:, retry_it: true)
+      if Time.now.to_i - @cache_timestamp > 3600
```

Comment:
I suggest exporting `3600` as a named constant, since we're duplicating the magic number (in `3601` form) in the tests. This applies to collections and institutions.

Reply:
I found to my surprise that you _can_ say `Timecop.travel(Time.now + described_class::CACHE_TTL + 1)`

Reply:
Excellent suggestion. Implemented in [75b14a2](https://github.com/hathitrust/holdings-backend/pull/388/commits/75b14a2de07ceef4d68292b13a395f3083ee1dea)